Katello/katello
# Fixes #30532 - CV publish with separate package env copy task

Currently blocked on https://pulp.plan.io/issues/7248

To test:
1. Set up pulp_rpm 3.5.0 and then apply this patch https://github.com/pulp/pulp_rpm/pull/1794 and restart pulp
2. Sync RHEL 8 Appstream repo
3. Create content view with any filter and publish.
4. Look at the pulp3 data with something like:
curl --cert /etc/pki/katello/certs/pulp-client.crt --key /etc/pki/katello/private/pulp-client.key https://`hostname`/pulp/api/v3/repositories/rpm/rpm/`997aa028-3467-47a5-95b3-d92dbaf43b1e`/versions/ | python -m json.tool
of the target repo to make sure all package environment were added to the latest version.
